docs/global-administration.md

```diff
@@ -1,42 +1,56 @@
 ---
-title: Global Administration
+title: Add User
```

Comment:
Why did the filename stay the same? Shouldn't it be changed to add-user to correlate with the title?